ddtrace/contrib/internal/botocore/patch.py

```diff
@@ -104,10 +104,14 @@ def _load_dynamodb_primary_key_names_for_tables() -> Dict[str, Set[str]]:
         "empty_poll_enabled": asbool(os.getenv("DD_BOTOCORE_EMPTY_POLL_ENABLED", default=True)),
         "dynamodb_primary_key_names_for_tables": _load_dynamodb_primary_key_names_for_tables(),
         "add_span_pointers": asbool(os.getenv("DD_BOTOCORE_ADD_SPAN_POINTERS", default=True)),
+        "payload_tagging_request": os.getenv("DD_TRACE_CLOUD_REQUEST_PAYLOAD_TAGGING", default=""),
+        "payload_tagging_response": os.getenv("DD_TRACE_CLOUD_RESPONSE_PAYLOAD_TAGGING", default=""),
+        "payload_tagging_max_depth": int(os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_MAX_DEPTH", 10)),
+        "payload_tagging_max_tags": int(os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_MAX_TAGS", 758)),
+        "payload_tagging_services": set(os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_SERVICES", default="s3,sns,sqs,kinesis,eventbridge").split(",")),
```

Comment:
This `.split` feels wrong - I can't find other examples of something like this in other `patch.py` files.

Reply:
Ah thanks, done!